src/query/config/src/config.rs

```diff
@@ -1647,6 +1647,11 @@ pub struct MetaConfig {
     #[serde(alias = "auto_sync_interval")]
     pub auto_sync_interval: u64,
 
+    /// try_config_endpoints_interval is the interval to try config endpoint when meta node endpoints is empty.
+    #[clap(long = "try-config-endpoints-interval", default_value = "10")]
+    #[serde(alias = "try_config_endpoints_interval")]
+    pub try_config_endpoints_interval: u64,
```

Comment:
Name to `refresh_meta_endpoints_interval` is better?

Reply:
we now have a `auto_sync_interval` config, which used to sync meta node endpoint internal, `refresh` means it will change endpoints, but this config is used to set internal use config endpoints if meta node is down.